res/css/views/dialogs/_HostSignupDialog.scss

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_HostSignupDialog {
+    min-height: 531px;
+    max-height: 600px;
+    width: 580px;
```

Comment:
These magical numbers should either be more dynamic or explained by a comment, `531` is especially odd.

Reply:
These are largely based on trying to get the content fit nicely  as per figma, ie not too little or much width. Might be able to get rid of these, will have a look, the design/implementation has lived quite a few times ;)

Reply:
So will the layout not work as well in other languages then?

Reply:
I don't think there are immediate plans to translate the EMS setup flow. If some other hosting provider wants to use this mechanism, they would need to design their setup wizard according to their needs.

Reply:
Having such a specific min-max height doesn't seem like it'll be very responsive for mobile etc, especially with a keyboard open though